topics/tutorials/kotlin.md

```diff
@@ -86,13 +86,13 @@ Plugins using the [Gradle Build System](gradle_build_system.md) use the [Kotlin
 
 See the <path>build.gradle</path> from [kotlin_demo](https://github.com/JetBrains/intellij-sdk-code-samples/tree/main/kotlin_demo) sample plugin:
 
-```groovy
+```kotlin
 ```
-{src="kotlin_demo/build.gradle"}
+{src="kotlin_demo/build.gradle.kts" lines="2-"}
 
 ### Use Kotlin for Gradle Build Scripts
 
-Gradle also supports using Kotlin in build scripts: <path>build.gradle.kts</path>.
+Gradle also supports using Kotlin in the build scripts: <path>build.gradle.kts</path>.
```

Comment:
Remove "the" again. Plural nouns _can_ be used with a definite article but they don't need it. The original version reads better.